components/ModSubmissionsEditFormFacility.vue

```diff
@@ -177,31 +177,15 @@
                 invalidInputErrorMessage="Invalid Longitude"
```

Comment:
Not part of this change, but we should avoid using raw text in the template like here.